src/pages/ProfilePage.js

```diff
@@ -280,12 +273,12 @@ export default compose(
         personalDetails: {
             key: ONYXKEYS.PERSONAL_DETAILS_LIST,
         },
-        loginList: {
-            key: ONYXKEYS.LOGIN_LIST,
-        },
         isLoadingReportData: {
             key: ONYXKEYS.IS_LOADING_REPORT_DATA,
         },
+        session: {
```

Comment:
Add session to props

Reply:
Thank you. Done. Apologies for missing basic things in haste.